### PR TITLE
Update Remind Window, create UiUtil, add Chester's PPP (WIP)

### PR DESCRIPTION
**Pull request by ChesterSim**

### CODE REVIEW

File: docs/team/chester.adoc [hunk 1]
```diff
@@ -0,0 +1,77 @@
+= Chester Sim - Project Portfolio
```
Comment (alcen) [inline docs/team/chester.adoc]: File name should be chestersim.adoc (according to GitHub username)

**Review by alcen — CHANGES_REQUESTED** [review]
Be sure to rename your PPP!

File: docs/team/chester.adoc (hunk 1 above)
Comment (ChesterSim) [reply docs/team/chester.adoc]: Thanks for the catch!

**Review by alcen — APPROVED** [review]
mascotApproval

**Pull request merged**